{"uid":"33034fc73ee0c600","name":"An RTP is sent through POSTE API","fullName":"tests.test_debtor_service_provider_availability#test_send_rtp_to_poste","historyId":"9ee43ed2ff7f9ef504e1b4a4edc013a4","time":{"start":1751267318577,"stop":1751267320578,"duration":2001},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1751267281654,"stop":1751267281710,"duration":56},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"story","value":"Service provider sends an RTP to POSTE directly"},{"name":"feature","value":"RTP Send"},{"name":"tag","value":"poste"},{"name":"tag","value":"happy_path"},{"name":"tag","value":"send"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_debtor_service_provider_availability"},{"name":"host","value":"pkrvmbietmlfzoi"},{"name":"thread","value":"2594-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_debtor_service_provider_availability"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":2,"broken":58,"skipped":0,"passed":3,"unknown":0,"total":63},"items":[{"uid":"c426bb006934cd45","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/417//#testresult/c426bb006934cd45","status":"passed","time":{"start":1751180869937,"stop":1751180870779,"duration":842}},{"uid":"92792e8a828b6172","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/416//#testresult/92792e8a828b6172","status":"passed","time":{"start":1751094452513,"stop":1751094453454,"duration":941}},{"uid":"35fcf61f767c237b","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/415//#testresult/35fcf61f767c237b","status":"failed","statusDetails":"AssertionError: assert 400 == 201\n +  where 400 = <Response [400]>.status_code","time":{"start":1751008093938,"stop":1751008095046,"duration":1108}},{"uid":"65538275fb1ccf07","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/414//#testresult/65538275fb1ccf07","status":"failed","statusDetails":"AssertionError: assert 400 == 201\n +  where 400 = <Response [400]>.status_code","time":{"start":1750947026403,"stop":1750947027108,"duration":705}},{"uid":"3f2f901ea29601b6","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/413//#testresult/3f2f901ea29601b6","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='rtp.svil.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)')))","time":{"start":1750924199462,"stop":1750924199848,"duration":386}},{"uid":"738b21c84e069566","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/412//#testresult/738b21c84e069566","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='rtp.svil.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)')))","time":{"start":1750921680589,"stop":1750921681034,"duration":445}},{"uid":"258a67a02043dc36","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/411//#testresult/258a67a02043dc36","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='rtp.svil.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)')))","time":{"start":1750856382363,"stop":1750856382752,"duration":389}},{"uid":"f3bccb7ad39d6285","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/408//#testresult/f3bccb7ad39d6285","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0557b9ae90>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750835302395,"stop":1750835437140,"duration":134745}},{"uid":"db0b91de8e344dd","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/407//#testresult/db0b91de8e344dd","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3e0039f4d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750780537389,"stop":1750780670777,"duration":133388}},{"uid":"808c7792ef20b62b","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/406//#testresult/808c7792ef20b62b","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0c6865ae90>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750779068183,"stop":1750779201593,"duration":133410}},{"uid":"1dc07602aba043b1","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/405//#testresult/1dc07602aba043b1","status":"broken","statusDetails":"OSError: Could not find the TLS certificate file, invalid path: /home/runner/work/rtp-platform-qa/rtp-platform-qa/config/cert.pem","time":{"start":1750777965692,"stop":1750777965693,"duration":1}},{"uid":"dc84809653bf1532","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/404//#testresult/dc84809653bf1532","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7895132e90>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750756725988,"stop":1750756860224,"duration":134236}},{"uid":"970bf40937eea695","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/403//#testresult/970bf40937eea695","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fd1c0c560d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750748875725,"stop":1750749009662,"duration":133937}},{"uid":"6f0f6b4e520793f6","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/402//#testresult/6f0f6b4e520793f6","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7e56c2e0d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750684871241,"stop":1750685007600,"duration":136359}},{"uid":"57a799f2d9f4bee9","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/401//#testresult/57a799f2d9f4bee9","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f340528f250>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750662510703,"stop":1750662646306,"duration":135603}},{"uid":"40885e7ced652fc9","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/400//#testresult/40885e7ced652fc9","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f920765a0d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750576084541,"stop":1750576220836,"duration":136295}},{"uid":"257cdf92bb30591c","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/399//#testresult/257cdf92bb30591c","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f99b804f250>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750489651935,"stop":1750489787142,"duration":135207}},{"uid":"bdc87548abc6f41b","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/398//#testresult/bdc87548abc6f41b","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f49405ef250>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750403245452,"stop":1750403381968,"duration":136516}},{"uid":"957d47383e84a297","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/397//#testresult/957d47383e84a297","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f4b15363250>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750335017992,"stop":1750335154023,"duration":136031}},{"uid":"b9d30daf6e270050","reportUrl":"https://pagopa.github.io/rtp-platform-qa/functional/396//#testresult/b9d30daf6e270050","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='rtp.coll.poste.it', port=446): Max retries exceeded with url: /iso20022/api/v1/sepa-request-to-pay-requests (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fef93a720d0>, 'Connection to rtp.coll.poste.it timed out. (connect timeout=5000)'))","time":{"start":1750329658096,"stop":1750329794563,"duration":136467}}]},"tags":["poste","send","happy_path"]},"source":"33034fc73ee0c600.json","parameterValues":[]}